app/workers/downstream_discard_draft_worker.rb

```diff
@@ -46,11 +46,11 @@ def assign_attributes(attributes)
   def assign_backwards_compatible_content_item(attributes)
     if attributes[:locale]
       @locale = attributes[:locale]
-      @web_content_item = Queries::GetWebContentItems.for_content_store(content_id, locale, true)
+      @edition = Queries::GetEditionForContentStore.(content_id, locale, true)
     else
       content_item_id = attributes[:live_content_item_id]
-      @web_content_item = content_item_id ? Queries::GetWebContentItems.find(content_item_id) : nil
-      @locale = web_content_item.try(:locale)
+      @edition = content_item_id ? Edition.find(content_item_id) : nil
+      @locale = edition.try(:locale)
```

Comment:
doh! sorry I misunderstood the code (and I'm pretty sure it was me that introduced the try in the first place)